### PR TITLE
[ZEPPELIN-5699] Trim space in ConfInterpreter

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

Trim the space for the key & value of ConfInterpreter. Unit test is added.

### What type of PR is it?
[Bug Fix ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5669

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/ConfInterpreter.java [hunk 1]
```diff
@@ -67,6 +67,9 @@ public InterpreterResult interpret(String st, InterpreterContext context)
       finalProperties.putAll(getProperties());
       Properties newProperties = new Properties();
       newProperties.load(new StringReader(st));
+      for (String key : newProperties.stringPropertyNames()) {
```
[the remark this answers is not in the document]
Comment (zjffdu) [reply zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/ConfInterpreter.java:70]: This is the test https://github.com/apache/zeppelin/pull/4336/files#diff-b266d633c5b6f9cdfbffde2c0a90e2d6f021b75bf2815c38c3644042cdd3b2a0R73

[the remark this answers is not in the document]
Comment (Reamer) [reply zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/ConfInterpreter.java:70]: I mean the following situation.
`newProperties.load(new StringReader(st));` loads the following property:
```
'myproperty       ': 'test'
```
Afterwards `newProperties` contains the following values:
```
'myproperty       ': 'testValue'`
'myproperty': 'testValue'
```
Now you have two property keys, one with spaces and one without.
Is this intentional?

[the remark this answers is not in the document]
Comment (zjffdu) [reply zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/ConfInterpreter.java:70]: Thanks, @Reamer , you are right, keys without trim should not be in the properties, I have updated this PR, test is covered.

**Review by Reamer — APPROVED** [review]
LGTM

**Pull request closed without merging**